modules/core/keeper/msg_server.go

```diff
@@ -79,6 +79,8 @@ func (k Keeper) UpgradeClient(goCtx context.Context, msg *clienttypes.MsgUpgrade
 }
 
 // SubmitMisbehaviour defines a rpc handler method for MsgSubmitMisbehaviour.
+// Warning: DEPRECATED
```

Comment:
Is that what you meant @colin-axner ?

Reply:
I think it would be good to add a deprecation notice to the proto files for `MsgSubmitBehaviour`.

Unfortunately protobuf only seems to have the field option [`[deprecated = true]`](https://developers.google.com/protocol-buffers/docs/proto3#options), there isn't any concrete deprecation flags for an entire message type.

Reply:
Yeah great point, thanks.

Reply:
Done: see https://github.com/cosmos/ibc-go/pull/1188#discussion_r840666627